composer.json

```diff
@@ -86,14 +85,16 @@
         "symfony/security-bundle": "^5.4 || ^6.4 || ^7.0",
         "twig/twig": "^3.0",
         "vimeo/psalm": "^5.20",
-        "willdurand/hateoas-bundle": "^2.0"
+        "willdurand/hateoas-bundle": "^2.0",
+        "winzou/state-machine-bundle": "^0.6.2"
     },
     "conflict": {
         "doctrine/orm": "<2.18 || ^3.0",
         "doctrine/doctrine-bundle": "<2.0 || ^3.0",
         "friendsofsymfony/rest-bundle": "<3.0",
         "jms/serializer-bundle": "<3.5",
         "willdurand/hateoas-bundle": "<2.0 || ^2.6",
+        "winzou/state-machine-bundle": "<0.6.2",
```

Comment:
Is this necessary if we have specified the minimum version as `"winzou/state-machine-bundle": "^0.6.2"`?

Reply:
We have, but only in dev-requirement, so to be sure in the end application that the proper and supported version will be installed, this conflict is needed